src/ring_theory/polynomial/hermite.lean

```diff
@@ -50,4 +51,80 @@ begin
   simp only [map_one, mul_one, derivative_one, sub_zero]
 end
 
+/-!
+### Lemmas about `polynomial.coeff`
+
+## Main definitions
+
+* `polynomial.coeff_hermite_of_odd_add`: for `n`,`k` where `n+k` is odd, `(hermite n).coeff k` is
+  zero.
+* `polynomial.monic_hermite`: for all `n`, `hermite n` is monic.
```

Comment:
These should go in the top module docstring; and should be titled "results" as these are not definitions.